### PR TITLE
TRIB-116

### PR DESCRIPTION
**Pull request by Jayj30pro**
Added new endpoint /api/permissions/user-roles-list to display user all user roles by name instead of number

### CODE REVIEW

File: src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java [hunk 1]
```diff
@@ -3,24 +3,31 @@
...
     UserRoleMapService userRoleMapService;
 
+    @Autowired
+    UserRoleService userRoleService;
+
     @Autowired
     UserService userService;
 
-    @GetMapping("/users")
+    @RequestMapping(value = { "/api/permissions/users"}, method=RequestMethod.GET)
```
Comment (haxwell) [inline src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java:30]: The endpoint mapping before is the correct way of doing it (GetMapping). Just FYI, we made a system wide change to that way of doing things a couple weeks ago.

File: src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java [hunk 2]
```diff
@@ -39,7 +46,7 @@ public ResponseEntity<Iterable<User>> getAllUsers() {
         }
     }
 
-    @GetMapping("/user-roles")
+    @RequestMapping(value = { "/api/permissions/user-roles"}, method=RequestMethod.GET)
```
Comment (haxwell) [inline src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java:49]: The endpoint mapping before is the correct way of doing it (GetMapping). Just FYI, we made a system wide change to that way of doing things a couple weeks ago.

File: src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java [hunk 3]
```diff
@@ -60,7 +78,7 @@ public ResponseEntity<Boolean> addPermissions(@RequestBody @Valid PermissionsReq
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(rtn);
         }
     }
-    @DeleteMapping
+    @RequestMapping(value = { "/api/permissions" }, method= RequestMethod.DELETE)
```
Comment (haxwell) [inline src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java:81]: The endpoint mapping before is the correct way of doing it (DeleteMapping). Just FYI, we made a system wide change to that way of doing things a couple weeks ago.

**Review by haxwell — CHANGES_REQUESTED** [review]
This PR looks good, other than the endpoint definition changes you made.. roll those back, and we should be good.

File: src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java (hunk 1 above)
Comment (Jayj30pro) [reply src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java]: Done

**Review by haxwell — APPROVED** [review]

**Pull request merged**